docs/Configuration.md

```diff
@@ -263,6 +254,39 @@ Custom Annotation names may also be configured using ConfigMap Data Values.
 
 Note: The requirement to label the build with `app.kubernetes.io/name=<app_name>` for the annotated Builds applies.
 
+#### An example workflow for an OpenShift binary build:
+
+```
+NS=mongo-persistent
+NAME=python-binary-build
+
+oc create namespace "${NS}"
+
+oc new-build python --name="${NAME}" --binary=true -n "${NS}"  -l "app.kubernetes.io/name=${NAME}"
+oc start-build "bc/${NAME}" --from-file=./app.py --follow -n "${NS}"
+
+
+oc -n "${NS}" annotate build "${NAME}-1" --overwrite \
```

Comment:
prior to that command ensure build is created:

```
$ oc get builds -n "${NS}"
NAME                  TYPE      FROM          STATUS     STARTED        DURATION
python-binary-build-1   Source    Binary    Complete   53 seconds ago   50s

```

Reply:
thanks, fixed